inst/joss_paper/paper.Rmd

```diff
@@ -0,0 +1,115 @@
+When using such models for real life applications, it is often crucial to understand why a certain set of features lead 
+to exactly a specific prediction.
+Most machine learning models are however so complicated and hard to understand that they are often viewed as 
+"black-boxes" producing output when provided some input.
+
+Shapley values (@Shapley53) is a concept from cooperative game theory used to fairly distribute a joint payoff among the
+cooperating players. 
+@kononenko2010efficient and later @lundberg2017unified proposed to use the Shapley value framework to explain 
+predictions by distributing the prediction value on the input features. 
+Unfortunately, established methods and implementations for explaining predictions with Shapley values like Shapley 
+Sampling Values (@vstrumbelj2014explaining), SHAP/Kernel SHAP (@lundberg2017unified), and to some extent TreeSHAP 
+(@lundberg2018consistent), assume that the features are independent when approximating the Shapley values for prediction
+explanation. 
+This `R`-package implements methodology proposed by @aas2019explaining to explain predictions by accounting for the 
+dependence between the features, resulting in significantly more accurate approximations to the Shapley values. 
+
+
+
+# Implementation
+
+The package implements a variant of the Kernel SHAP (@lundberg2017unified) methodology for efficiently dealing with the 
+combinatorial problem related to Shapley value formula. 
+The main methodological contribution of @aas2019explaining is three different methods to estimate certain conditional 
+expectation quantities, referred to as the `empirical`, `gaussian` and `copula` approach. Additionaly, the user has
+the option of combining the three approaches. 
+The implementation supports explanation of the following models natively: `stats::lm`, `stats::glm`, `ranger::ranger`, 
+`mgcv::gam` and `xgboost::xgboost`. 
+Moreover, the package supports explanation of custom models by supplying two simple additional class functions.
+
+The user interface in the package has largely been adopted from the `R`-package `lime` (@limeRpackage). 
+The user first sets up the explainability framework for the model to explain with the `shapr` function. 
+Then the output from `shapr` is provided to the `explain` function, along with the data to explain the prediction for
+and which method should be used to estimate the aforementioned conditional expectations.
+
+The majority of the code is plain `R`, while the most time consuming parts of the code has been coded in `C++` through 
+the `Rcpp` package for computational speed up. 
+Our implementation has some minor methodological improvements to the Kernel SHAP methodology as implemented in the 
+`Python` package `shap` (@shapPythonpackage). 
+In addition to our package's ability to account for the feature dependence (which the `shap` package does not), 
+basic tests suggest our implementation is about 3-4 times faster. 
```

Comment:
This sounds vague.

Reply:
Indeed, it is vague. I don't think we have the space to say what our methodological improvements really are. Ok with skipping the speedup thing and keeping the rest similar to how it is? @nikolase90

Reply:
Attempted to fix this in 16ff339/ d107cfa, by deleting/moving the relevant parts. What do you think @nikolase90 ?

Reply:
:+1: